### DISCUSSION TITLE
Add support for left-handed pointer devices

### DISCUSSION THREAD

**Discussion by dnkl** [YaLTeR/niri#365] [Ideas]
I use trackballs and mice with my left hand, and I'm used to swap the buttons. Many wayland compositors have a "left-hand" option that can be set on pointer/trackpad devices. The libinput function is [libinput_device_config_left_handed_set](https://smithay.github.io/input.rs/input_sys/fn.libinput_device_config_left_handed_set.html)

In Sway, the option is `input <id> left_handed enable|disable`.

In River, it's `riverctl input <id> left-handed enabled|disabled`.

**Comment by YaLTeR** [comment]
Made an issue

**Comment by YaLTeR** [comment]
Adding libinput props is fast so I just went ahead and added it c9d90afe591ea133955409b05bb9039e622da9e5

**Reply by dnkl** [reply]
Awesome, that was super fast :) thanks!
